### DISCUSSION TITLE
Manuelle Fahrten

### DISCUSSION THREAD

**Discussion by srype** [Traewelling/traewelling#2034] [Ideas / Feature Requests]
Ein Manuelles Einfügen von Fahrten (mit Zwischenhalten) die nicht über die Fahrplan API kommen wäre eine coole Sache. So könnte man auch Sonderfahrten aller Art hinzufügen. Das ganze natürlich ohne Punktewertung.

**Comment by HerrLevin** [comment]
We should also add the possibility to create taxi-checkins.

Maybe we could also add the possibility of adding other means of transport, e.g. bike, car or plane? (Needs discussion)

**Comment by MrKrisKrisu** [comment]
Zwischenstand:
Es gibt mittlerweile eine geschlossene Beta, wo ein paar User die Erstellung von manuellen Trips testen. Bisher funktioniert das ganz gut. Denke also, das könnte bald in Richtung Produktion gehen. :)

**Reply by MrKrisKrisu** [reply]
Manuelle Fahrten sind (mit Einschränkungen) jetzt möglich. Dazu einfach die experimentellen Funktionen in den Einstellungen aktivieren, dann ist dieser Link nutzbar:

https://traewelling.de/trip/create
